source/OctoVersion.Tests/WhenParsingASemanticVersion.cs

```diff
@@ -23,12 +23,12 @@ public void TheResultingVersionShouldBeCorrect(string input, SemanticVersion exp
 
         public static IEnumerable<object[]> TestCases()
         {
-            yield return new object[] { "", null };
-            yield return new object[] { "spit.spot", null };
-            yield return new object[] { "1.2.x", null };
-            yield return new object[] { "1.x.3", null };
-            yield return new object[] { "x.2.3", null };
-            yield return new object[] { "1..2", null };
+            yield return new object[] { "", null! };
+            yield return new object[] { "spit.spot", null! };
+            yield return new object[] { "1.2.x", null! };
+            yield return new object[] { "1.x.3", null! };
+            yield return new object[] { "x.2.3", null! };
+            yield return new object[] { "1..2", null! };
```

Comment:
This is _baffling_, but if the compiler says it wants this, who are we to disagree?

Reply:
"No really, this is not `null`, trust me, compiler".